types/mailgun-js/index.d.ts

```diff
@@ -74,6 +75,11 @@ declare namespace Mailgun {
             "recipient-variables"?: BatchSendRecipientVars;
         }
 
+        interface SendTemplateData {
```

Comment:
The interface should also have the following properties from `SendData`: `from`, `to`, `cc`, `bcc`, `subject`, `attachment` and `inline`.
Especially `to` would be important since it is required to send the mail.

Could we do something like the following?
```typescript
interface SendDataBase {
    from?: string;
    to: string | string[];
    cc?: string;
    bcc?: string;
    subject?: string;
    attachment?: AttachmentData | ReadonlyArray<AttachmentData>;
    inline?: AttachmentData | ReadonlyArray<AttachmentData>;
}

interface SendData extends SendDataBase {
    text?: string;
    html?: string;
}

interface SendTemplateData extends SendDataBase {
    template: string;
    [templateVariable: string]: string;
}
```